test/server/create-testcafe-test.js

```diff
@@ -90,7 +90,7 @@ describe('TestCafe factory function', function () {
     it('Should contain plugin testing, embedding utils and common runtime functions', function () {
         expect(createTestCafe.pluginTestingUtils).to.be.an.object;
         expect(createTestCafe.embeddingUtils).to.be.an.object;
-        expect(createTestCafe.Role).eql(commonAPI.Role);
-        expect(createTestCafe.ClientFunction).eql(commonAPI.ClientFunction);
+        expect(createTestCafe.Role).eql(exportableLib.Role);
```

Comment:
It doesn't relate to this PR changes, but this check look strange for me since we don't have `Role` for now. Actually this line checks that `undefined` equals `undefined`.